assets/panel/prefs.js

```diff
@@ -59,3 +59,4 @@ pref("devtools.debugger.features.event-listeners", false);
 pref("devtools.debugger.features.code-folding", false);
 pref("devtools.debugger.features.outline", true);
 pref("devtools.debugger.features.replay", false);
+pref("devtools.debugger.features.async-stepping", true);
```

Comment:
if we add `devtools.debugger.features.skip-pausing` we can land this work and then turn it on when the server is ready...